src/metabase/query_processor/middleware/process_userland_query.clj

```diff
@@ -95,6 +95,7 @@
        (dissoc :error :hash :executor_id :action_id :is_sandboxed :card_id :dashboard_id :pulse_id :result_rows :native))
    (dissoc result :cache/details)
    {:cached                 (boolean (:cached cache))
+    :updated_at             (:updated_at cache)
```

Comment:
should we be more explicit ? how about `cache_updated_at`

Reply:
Yeah I thought about that (and also some tests just failing on that) and moved value to `cached`. 😁